code/modules/antagonists/revolution/revolution.dm

```diff
@@ -305,7 +305,7 @@
 			var/list/datum/mind/promotable = list()
 			var/list/datum/mind/nonhuman_promotable = list()
 			for(var/datum/mind/khrushchev in non_heads)
-				if(khrushchev.current && !khrushchev.current.incapacitated() && !khrushchev.current.restrained() && khrushchev.current.client && khrushchev.current.stat != DEAD)
+				if(khrushchev.current && !khrushchev.current.incapacitated() && !khrushchev.current.restrained() && khrushchev.current.client && khrushchev.current.stat != DEAD && khrushchev.current.job != "Shaft Miner")
```

Comment:
1. this only runs after roundstart
2. this is awful snowflake, should check for z level of the mob or something

Reply:
#1 is the point